desktop/mac/release-notes.md

```diff
@@ -17,14 +17,43 @@ toc_max: 2
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
-## Docker Desktop 4.0.1
-2021-09-13
+## Docker Desktop 4.1.0
+2021-09-30
 
 > Download Docker Desktop
 >
 > [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
 > [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
 
+### New
+
+- **Preferences**
+  - New update section to manage updates 
+  - Compose V2 is now an option in General section
+- **Volume Management**: available for all users
+
+### Upgrades
+
+- [Compose V2](https://github.com/docker/compose/releases/tag/v2.0.0)
+- [Buildx 0.6.3](https://github.com/docker/buildx/releases/tag/v0.6.3)
+- [Kubernetes 1.21.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.5)
+- [Go 1.17.1](https://github.com/golang/go/releases/tag/go1.17.1)
+- [Alpine 3.14](https://alpinelinux.org/posts/Alpine-3.14.0-released.html)
+- [Qemu 6.1.0](https://wiki.qemu.org/ChangeLog/6.1)
+- Base distro to debian:bullseye
+
+### Bug fixes and minor changes
+
+- Fixed CVE Report GHSL-2021-058 by Github
```

Comment:
Do we have a URL that we can use for the CVE report?

Reply:
The link to the CVE is not public yet according to @fredericdalleau